hack/netpol-cyclonus/install-cyclonus.yml

```diff
@@ -0,0 +1,21 @@
+apiVersion: batch/v1
```

Comment:
Perhaps we should move this into `test/fuzzing`?

Reply:
This definitely deserves its own directory. Cyclonus does proper end-to-end testing of network policies, it's not a fuzzer, so I would put it in `test/cyclonus` or `test/networkpolicy` or similar.

Reply:
👍 👍 

Hopefully we can get cyclonus to doing something more similar to fuzzing in the not-too-distant future ? 😄   I think part of the calculus on where it goes next it what provides the most value to cilium (regression tests?  conformance tests?  tests against randomly generated network policies?  something else ... 🤔 )  

Whatever does end up being the best route, I think the changes in this github workflow will be tiny: bump cyclonus to a new version, change the args passed in -- shouldn't be disruptive or complicated!

Anyway just 🤔 out loud here, happy to collab more on this!  (Not looking to just do a drive-by and dump some quickly out-of-date tests on you 😄 😄 )

Reply:
@mattfenwick would you mind moving these files to a `test/netpol-cyclonus` directory so they're alongside the other tests? Thanks!